### DISCUSSION TITLE
Error en Submit

### DISCUSSION THREAD

**Discussion by efsiul** [rramosp/ai4eng.v1#237] [01 - INTRODUCCIÓN]
Buenas tardes, 

Como se indicó anteriormente al docente a través de correo, soy uno de los estudiantes a los que se ajustó hasta la semana pasada la materia "Inteligencia Artificial para Ingeniería y las Ciencias". Por ello, tengo entendido que hasta mañana hay plazo para entregar el laboratorio 1 y 2. Al tratar de enviar los diferentes puntos, ejecutando la función que hace submit, me sale el siguiente error: "error: end_date (set session.debug=True for tracebacks)"
Espero me colaboren con la solución. 

Gracias.

**Comment by rramosp** [comment]
no te debería de pasar pues la fecha es hasta mañana ... inténtalo de
nuevo  y mándanos un pantallazo actualizado

On Thu, Mar 2, 2023 at 12:01 PM Luis Felipe Cadavid Chica <
***@***.***> wrote:

> Buenas tardes,
>
> Como se indicó anteriormente al docente a través de correo, soy uno de los
> estudiantes a los que se ajustó hasta la semana pasada la materia
> "Inteligencia Artificial para Ingeniería y las Ciencias". Por ello, tengo
> entendido que hasta mañana hay plazo para entregar el laboratorio 1 y 2. Al
> tratar de enviar los diferentes puntos, ejecutando la función que hace
> submit, me sale el siguiente error: "error: end_date (set
> session.debug=True for tracebacks)"
> Espero me colaboren con la solución.
>
> Gracias.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/237>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZII7TUP2QUCRMOILPVLW2DHALANCNFSM6AAAAAAVNUKQTM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by efsiul** [comment]
Lo volvi a intentar, pero consigo el mismo error. Adjunto captura. 
 
![Captura desde 2023-03-02 15-06-23](https://user-images.githubusercontent.com/47805420/222540816-2b052923-2217-48be-89ee-02efa8caf26a.png)

**Comment by rramosp** [comment]
luis felipe, es que está sometiéndolo en la sesión del semestre pasado. te
tienes que registrar en este semestre siguiendo las instrucciones del curso.

una vez que te registres haznos saber para eliminarte de la sesión del
semestre pasado



On Thu, Mar 2, 2023 at 3:09 PM Luis Felipe Cadavid Chica <
***@***.***> wrote:

> Lo volvi a intentar, pero consigo el mismo error. Adjunto captura.
>
> [image: Captura desde 2023-03-02 15-06-23]
> <https://user-images.githubusercontent.com/47805420/222540816-2b052923-2217-48be-89ee-02efa8caf26a.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/237#discussioncomment-5182872>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIOIMAUDLEP3PJSSEYLW2D47VANCNFSM6AAAAAAVNUKQTM>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by efsiul** [reply]
Ya se hizo profesor, ya me registré en este semestre

**Comment by rramosp** [comment]
hola luis felipe, te tienes que inscribir en el de la página de información
del curso de la udea, no del abierto.

On Thu, Mar 2, 2023 at 3:38 PM Luis Felipe Cadavid Chica <
***@***.***> wrote:

> Ya se hizo profesor, ya me registré en este semestre
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/237#discussioncomment-5183080>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIJ2HDBXSSSKM3HM5ZTW2EALTANCNFSM6AAAAAAVNUKQTM>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by Felipe-RA** [comment]
Saludos @efsiul 

Para ahondar un poco más en lo que dice el profesor raúl, recuerda que debes seguir las indicaciones específicas de la versión UDEA, a las cuales puedes acceder tal  como te muestro abajo
<img width="257" alt="image" src="https://user-images.githubusercontent.com/49454068/222622753-cf9988d9-cfed-4884-8826-d118e66cb42f.png">

<img width="626" alt="image" src="https://user-images.githubusercontent.com/49454068/222622835-e53e74d5-ddae-470c-afda-77d73c313956.png">

¡Danos un toque cuando hagas esto, hasta pronto!

**Comment by DanielEstradaG** [comment]
Hola, Yo estoy teniendo problemas con las submission. ME da este error: 
![Captura](https://github.com/rramosp/ai4eng.v1/assets/160770264/5552976e-272a-47ad-bcd1-47c44fb9a1cd).
También en la parte del código donde debería ver el resultado, no lo imprime: 
![Captura2](https://github.com/rramosp/ai4eng.v1/assets/160770264/7f31093a-464c-4f3d-b2d8-b45433044d32)

Gracias

**Reply by Felipe-RA** [reply]
Saludos @DanielEstradaG 

El error, en la penúltima linea nos indica que es un NameError

Esto significa que hay problemas con el nombre de algo, especificamente con el nombre de np, que no esta importado dentro de la definición de función.

Recuerda poner todos los imports dentro de la función que te piden hacer para evitar estos problemas, no vale con tenerlos por fuera en alguna otra parte del notebook
